x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/Version.java

```diff
@@ -16,37 +16,20 @@
 import java.util.jar.JarInputStream;
 import java.util.jar.Manifest;
 
-public class Version {
+public class Version extends org.elasticsearch.xpack.sql.proto.Version {
```

Comment:
Functionally, this class is not another `Version` but a consumer of `Version`.
It has only `isServerCompatible` and `jdbcMajor/Minor` (which were added to this class since it was the only one but could be moved somewhere else).

Try refactoring it to something else either a consumer:

`ClientVersion(Version) { .. }`
or if there's no state, utility methods - ClientVersion/VersionUtils.isServerCompatible(version)`

Reply:
Sure. I've turned that class into a `ClientVersion` utility class.